package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@ni/javascript-styleguide",
+  "version": "1.0.0",
```

Comment:
I've seen some projects start with a 0.*.* version number to indicate the repo is in a prerelease state. Any objection to following that pattern until we've pulled in our first set of rule configurations?

Reply:
Yes, good idea. I usually use npm's semver syntax alpha and beta, because it works with any version.